openapi/components/requestBodies/CreditTransactionRequest.yaml

```
@@ -1,6 +1,124 @@
+        websiteId:
+          $ref: ../schemas/WebsiteId.yaml
+        customerId:
+          $ref: ../schemas/CustomerId.yaml
+        currency:
+          $ref: ../schemas/CurrencyCode.yaml
+        amount:
+          description: Amount of the transaction.
+          type: number
+          format: double
+          example: 97.97
+        invoiceIds:
+          description: Array of invoice IDs.
+          nullable: true
+          type: array
+          items:
+            $ref: ../schemas/ResourceId.yaml
+        paymentInstruction:
+          description: |-
+            Payment instruction for the purchase.
+            If this value is not supplied,
+            the customer's default payment instrument is used.
+          allOf:
+            - $ref: ../schemas/PaymentInstruction.yaml
+        billingAddress:
+          description: |-
+            Billing address.
+            If this value is not supplied,
+            the billing address associated with the payment instrument is used.
+            If no billing address is associated with the payment instrument,
+            the customer's billing address is used.
+          nullable: true
+          type: object
+          allOf:
+            - $ref: ../schemas/Contact/ContactObject.yaml
+        requestId:
+          description: |-
+            Use this field to prevent duplicate transaction requests that may occur within a short period of time.
+            If a duplicate request is sent with the same `requestId`,
+            it is ignored to prevent double-billing.
+            This value must be unique within a 24-hour period.
+
+            > **Important:** This field is recommended.
+          type: string
+          nullable: true
+          maxLength: 50
+          pattern: '^[\-\w]+$'
+          example: 44433322-2c4y-483z-a0a9-158621f77a21
+        gatewayAccountId:
+          description: |-
+            ID of the gateway account.
+            Rebilly selects the payment gateway account for the transaction based on transaction properties and the rules configuration of the `gateway-account-requested` event.
+            To prevent Rebilly from making the gateway account selection,
+            supply a gateway account ID in this field.
+            Only use this field if you intend to override the settings.
+          nullable: true
+          type: string
+          maxLength: 50
+          example: gw_acc_0YVCXMF26DDNKAERE5NW727S34
+        description:
+          nullable: true
+          description: Payment description.
+          type: string
+          maxLength: 255
+        notificationUrl:
+          nullable: true
+          description: |-
+            URL where a server-to-server `POST` notification is sent.
+            This notification is sent when the transaction result is finalized after a timeout or an offsite interaction.
+
+            Do not trust this notification alone,
+            complete a `GET` request to confirm the result of the transaction.
```

nitpick: 'trust this notification alone' is ambiguous 
```suggestion
            Do not interpret this notification as a confirmation,
            complete a `GET` request to confirm the result of the transaction.
```